### PR TITLE
feat(kit)!: `Flag` & `InputPhoneInternational` use new svg icons of country flags

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

File: projects/kit/components/input-phone-international/input-phone-international.style.less [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 .t-flag {
     width: 1.75rem;
-    height: 1.25rem;
+    height: 1.75rem;
+    border-radius: 50%;
```
Comment (nsbarsukov) [inline projects/kit/components/input-phone-international/input-phone-international.style.less:23]: You can already review this PR.
But don't merge. I am waiting for Sergey review

File: projects/core/pipes/flag/test/flag.pipe.spec.ts [hunk 2]
```diff
@@ -20,10 +20,10 @@ describe('tuiFlagPipe', () => {
     });
 
     it('calculates flag path from TUI_SVG_OPTIONS & isoCode', () => {
-        expect(pipe.transform('RU')).toBe('path/RU.png');
+        expect(pipe.transform('RU')).toBe('path/flags/RU.svg');
     });
 
     it('calculates flag path from isoCode', () => {
-        expect(pipe.transform('AD')).toBe('path/AD.png');
+        expect(pipe.transform('AD')).toBe('path/flags/AD.svg');
```
Comment (waterplea) [inline projects/core/pipes/flag/test/flag.pipe.spec.ts:27]: Are we sure we need a subfolder for flags? 🤔

File: projects/legacy/components/input-phone-international/input-phone-international.style.less [hunk 3]
```diff
@@ -42,14 +39,13 @@
...
-}
+    width: 1.75rem;
+    height: 1.75rem;
+    border-radius: 50%;
 
-.t-country-item-flag {
-    width: @flag-width;
-    height: @flag-height;
+    &_select {
+        margin-left: -@space * 2;
```
Comment (waterplea) [inline projects/legacy/components/input-phone-international/input-phone-international.style.less:47]: Let's drop `@space` var, it's useless

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.style.less (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/input-phone-international/input-phone-international.style.less:23]: Done

**Pull request merged**